vendor/.editorconfig

```diff
@@ -0,0 +1 @@
+root = true
```

Comment:
`make clean` will delete this... can update the `vendor` target to create it...

```
vendor: $(VENDOR)
	echo "root = true" > vendor/.editorconfig
```

Reply:
Fixed